config/opensearch_dashboards.yml

```diff
@@ -223,3 +223,12 @@
 # Set the value of this setting to true to start exploring wizard
 # functionality in Visualization.
 # wizard.enabled: false
+
+# Set the value of this setting to true to enables the experimental multiple data source
+# support feature. Use with caution.
+#data_source.enabled: false
```

Comment:
nit: For consistency, can we add a space between the comment delimiter and the config key?

Reply:
I think the convention in this file is:
* for comments/explanations, put a space between comment content and comment delimiter `#` 
* for sample config key/settings, no space between config key and comment delimiter `#`
So that it is easy for people to differentiate comment/explanations vs. sample configs